client/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
```

Comment:
Do you want to make it off-line app?
if not so I think no needs for `serviceWorker`

Reply:
I think it's good to try and make it a progressive app.